xmtp_proto/gen_protos.sh

```diff
@@ -0,0 +1,10 @@
+if ! cargo install --list | grep "protoc-gen-prost-crate" > /dev/null; then
```

Comment:
Would be nicer if we had a `dev` folder in the repo, to match the convention in other xmtp projects